ci/openshiftio-appdev-docs/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+launcher.template_url.latest=https\://raw.githubusercontent.com/openshiftio/launchpad-templates/v5/openshift/launchpad-template.yaml
```

Comment:
Thanks for catching that. I don't know how that ended up in there. I forgot to rebase, probably.